website/docs/components/table/partials/code/how-to-use.md

```diff
@@ -1,11 +1,14 @@
-This component takes advantage of the `sort-by` helper provided in [ember-composable-helpers](https://github.com/DockYard/ember-composable-helpers). While some of the examples provided on our documentation use some of the other helpers provided in this addon, they are not required to use Helios. Read the [configuration information](https://github.com/DockYard/ember-composable-helpers#configuration) provided by the addon if you wish to customize which helpers are included in your own app.
-
 ## How to use this component
 
+!!! Info
+
+This component takes advantage of the `sort-by` helper provided by [ember-composable-helpers](https://github.com/DockYard/ember-composable-helpers).
+
+!!!
```

Comment:
I want this information up-front for a few reasons; one is that many apps are already this addon in their applications. So the idea is, this thing is probably already familiar to you. But there was other feedback that the previous iteration felt too wordy, so I tried to strip it down to the bare minimum. Does that help clarify? WDYT?

Reply:
I like the stripped-down version. ✨ I'm more curious about its placement in the overall hierarchy of the docs because it's only needed for sortable tables, correct? If so, it feels like there's a more natural fit for this content within the section on sortable tables.

Reply:
@MelSumner not a blocker, but I strongly agree with @heatherlarsen: this is too prominent, and I think should be moved under the sortable tables examples where is more in context